test/IDE/complete_protocol_typealias.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-ide-test -batch-code-completion -source-filename %s -filecheck %raw-FileCheck -completion-output-dir %t
+
+protocol MyProto {
+  typealias Content = Int
```

Comment:
Can you add a `typealias` with generic arguments? e.g. `typealias Storage<T> = Array<T>`. I don't this your change affects this case, but still.

Reply:
Added a test case. Seems to not be an issue.